include/bluetooth/services/fast_pair/fmdn.h

```diff
@@ -468,6 +475,65 @@ struct bt_fast_pair_fmdn_info_cb {
  */
 int bt_fast_pair_fmdn_info_cb_register(struct bt_fast_pair_fmdn_info_cb *cb);
 
+/** @brief Motion detector callback structure.
```

Comment:
For consistency with the DULT API, please move the motion detection API just below the FMDN ring API (right after `bt_fast_pair_fmdn_ring_state_update`). Motion detection is kind of dependent on the sound/ring features so it may be a good idea to group them close to each other